src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp

```diff
@@ -261,6 +261,34 @@ START_SECTION((void load(const String & filename, std::vector<AbsoluteQuantitati
 
 END_SECTION
 
+START_SECTION(void store(const String & filename, const std::vector<AbsoluteQuantitationMethod> & aqm_list) const)
+  AbsoluteQuantitationMethodFile aqmf;
+  vector<AbsoluteQuantitationMethod> aqm_list1, aqm_list2;
+  aqmf.load(OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationMethodFile_1.csv"), aqm_list1);
+  aqmf.store(OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationMethodFile_2.csv"), aqm_list1);
+  aqmf.load(OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationMethodFile_1.csv"), aqm_list2);
+  TEST_EQUAL(aqm_list1.size(), aqm_list2.size())
+  for (Size i = 0; i < aqm_list1.size(); ++i)
```

Comment:
if `AbsoluteQuantitationMethod` had an `operator==` then you could just test for equality. But this is fine too...

Reply:
I will implement this in the coming days